zellij-server/src/panes/csi.rs

```diff
@@ -0,0 +1,5 @@
+/// A trait for values which have a csi-representation
+pub trait Csi {
```

Comment:
Is there a reason the trait is needed at this point? Do you need it for future development?

Reply:
No, I don't need it for the future development, it's just in my opinion suitable to create a `trait` for this since it emphasizes the context of `get_csi_str`. I can remove it if you want.

Reply:
OK, I got it. 

However, this may be a slightly different topic, but as you know, in zellij, we use a lot of CSI code in various crates as well as `zellij-server`. 

So I think the end goal of this refactoring should be to provide all the CSI code from `zellij-utils`. And if we provide a variety of CSI codes, this trait will definitely make sense. (Of course, this is not to say that this PR covers all of this.)

Anyway, personally, for now, it seems like a good idea to refactor with less code.

Reply:
Alright, I'll move it back then.

Reply:
Done